CHANGELOG.md

```diff
@@ -5,19 +5,24 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## Unreleased
+## [Unreleased]
 
-## [2.10.0] - 10-18-2023
+## [2.10.1] - 2023-10-31
+### Fixed
+- Update delete_v2_sessions function to use template-aware database keys (so that it actually
```

Comment:
tenant-aware rather than template aware.

Reply:
Thank you!